### PR TITLE
Sorting stale requests

### LINKED ISSUE #4362: CCTN 47
**Channel Name:** CCTN
**Country:** Philippines
**Language:** Filipino/Cebuano
**Website:** http://cctn47.com/
**Stream URL (optional):** http://122.55.252.134:8443/live/bba5b536faeacb9b56a3239f1ee8e3b3/1.m3u8
**Notes (optional):** Catholic TV station from Cebu, Philippines

### LINKED ISSUE #4363: CLTV 36
**Channel Name:** Central Luzon Television
**Country:** Philippines
**Language:** Filipino/Kapampangan
**Website:** http://cltv36.tv/
**Stream URL (optional):** https://tna5.bozztv.com/cltv36/index.m3u8
**Notes (optional):** TV station based in Pampanga, Philippines. The link is from ssh101, and it can use a referrer.

### LINKED ISSUE #4384: Kapamilya Channel
**Channel Name:** Kapamilya Channel
**Country:** Philippines
**Language:** Tagalog
**Website:** https://ent.abs-cbn.com/articles-news/kapamilya-channel-livestream-12685
**Stream URL (optional):** xxx
**Notes (optional):** Replacement of ABS-CBN

### LINKED ISSUE #4387: DD News
**Channel Name:** DD News
**Country:** IN
**Language:** hindi
**Website:** https://ddnews.gov.in/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

### LINKED ISSUE #4415: EWTN Español
**Channel Name:** EWTN Español
**Country:** US
**Language:** Spanish
**Website:** https://www.ewtn.com/es/tv/en-vivo
**Stream URL (optional):** https://cdn3.wowza.com/1/SmVrQmZCUXZhVDgz/b3J3MFJv/hls/n8345n84/2728/chunklist.m3u8
**Notes (optional):** I found the religious LATAM and Spain channel this link from Las Nazarenas peruvian website (https://www.nazarenas.tv/canalcatolico). However, I am not sure if it is operational with only one link, so I made this issue for you to confirm if I am correct.

### LINKED ISSUE #4578: Fox 9 Minneapolis (KMSP)
**_Channel Name:_** Fox 9 Minneapolis (KMSP)
**_Country:_** USA
**_Broken Link (from playlist):_** https://lnc-kmsp-fox-aws.tubi.video/index.m3u8
**_Possible Replacement (optional):_** xxx
**_Notes (optional):_** stream is repeating 10 second loop from a couple is days ago.

### LINKED ISSUE #4579: FOX Chicago News
**_Channel Name:_** FOX 32 Chicago IL (WFLD)
**_Country:_** USA
**_Broken Link (from playlist):_** https://lnc-wfld-fox-aws.tubi.video/index.m3u8
**_Possible Replacement (optional):_** xxx
**_Notes (optional):_** Same Issue to 4578.

### LINKED ISSUE #4607: Bang Bang
Channel Name: Bang Bang 
Country: Albanian 
Language: Shqip
Website: https://digitalb.al/

### LINKED ISSUE #4634: CW Seed
**Channel Name:** CW Seed
**Country:** USA
**Language:** English
**Website:** https://www.cwseed.com/
**Stream URL (optional):** https://cwseedlive.cwtv.com/ingest/playlist.m3u8 [USA IP ONLY]
**Notes (optional):** CW Seed is Internet Streaming Channel of CW. Not Actual CW Broadcast Feed.
(https://en.wikipedia.org/wiki/The_CW#CW_Seed)

### LINKED ISSUE #4655: Bang Bang
Channel Name: Bang Bang 
Country: Albanian 
Broken Link http://93.157.62.180/BangBang/index.m3u8

### LINKED ISSUE #4657: Deluxe Lounge HD
**Channel Name:** Deluxe Lounge HD
**Country:** Germany
**Language:** English
**Website:** www.deluxemusic.tv
**Stream URL (optional):** https://d46c0ebf9ef94053848fdd7b1f2f6b90.mediatailor.eu-central-1.amazonaws.com/v1/manifest/81bfcafb76f9c947b24574657a9ce7fe14ad75c0/live-prod/91fcad1e-54b1-4702-9ec1-22a379525281/0.m3u8
**Notes (optional):** Stream added from Samsung TV Plus. I tried to add but someone deleted.

### PR DESCRIPTION
**Pull request by Dum4G**
Closes iptv-org/iptv#4634
Closes iptv-org/iptv#4607
Closes iptv-org/iptv#4579
Closes iptv-org/iptv#4578
Closes iptv-org/iptv#4415
Closes iptv-org/iptv#4387
Closes iptv-org/iptv#4384
Closes iptv-org/iptv#4363
Closes iptv-org/iptv#4362
Closes iptv-org/iptv#4655
Closes iptv-org/iptv#4657

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -149,8 +149,8 @@ http://103.199.161.254/Content/ddmalayalam/Live/Channel(DDMalayalam)/index.m3u8
 http://103.199.161.254/Content/ddnational/Live/Channel(DDNational)/index.m3u8
 #EXTINF:-1 tvg-id="DDNational.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/MohlE5B.png" group-title="Entertainment",DD National (480p)
 https://m-c036-j2apps.s.llnwi.net/hls/0098.DDNational.in.m3u8
-#EXTINF:-1 tvg-id="DDNews.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/9ozOypJ.png" group-title="News",DD News (576p) [Offline]
-https://nicls1-lh.akamaihd.net/i/ddnews_1@409133/master.m3u8
+#EXTINF:-1 tvg-id="DDNews.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/9ozOypJ.png" group-title="News",DD News
+https://www.youtube.com/channel/UCKwucPzHZ7zCUIf7If-Wo1g/live
```
Comment (LaneSh4d0w) [inline channels/in.m3u:153]: ?

File: channels/al.m3u [hunk 2]
```diff
@@ -65,3 +65,7 @@ rtmp://live.tvkoha.tv:1935/live/koha/livestream
 http://ip.opoja.tv:1935/tvopoja/tvopoja/playlist.m3u8
 #EXTINF:-1 tvg-id="ZjarrTelevizion.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/UValLp1.png" group-title="",Zjarr Televizion (1080p)
 http://217.73.132.66:1935/live/Zjarr.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/TD4J83Q.png" group-title="Kids",Bang Bang
+http://93.157.62.180/BangBang/index.m3u8
```
[the remark this answers is not in the document]
Comment (Dum4G) [reply channels/al.m3u:69]: Was working yesterday, will be fixed on their end I guess

File: channels/in.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/in.m3u]: fixed

File: channels/ph.m3u [hunk 3]
```diff
@@ -29,3 +29,9 @@ http://stream.gmanews.tv/ioslive/livestream/chunklist.m3u8?wowzasessionid=693701
 https://iptv--iptv.repl.co/streamlink?url=https://www.youtube.com/c/TVMariaLIVE/live
 #EXTINF:-1 tvg-id="UNTV.ph" tvg-country="PH" tvg-language="" tvg-logo="https://i.imgur.com/CAlBD49.png" group-title="",UNTV (1080p) [Offline]
 https://cdn.untvweb.com/live-stream/untvweb.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="PH" tvg-language="Tagalog" tvg-logo="" group-title="",CCTN 47
+http://122.55.252.134:8443/live/bba5b536faeacb9b56a3239f1ee8e3b3/1.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="PH" tvg-language="Tagalog" tvg-logo="" group-title="",Kapamilya Channel
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCstEtN0pgOmCf02EdXsGChw/live
```
Comment (freearhey) [inline channels/ph.m3u:35]: image
The source is online:
image

Comment (LaneSh4d0w) [reply channels/ph.m3u:35]: Let me see.

File: channels/ph.m3u [hunk 4]
```diff
@@ -29,3 +29,9 @@ http://stream.gmanews.tv/ioslive/livestream/chunklist.m3u8?wowzasessionid=693701
...
+#EXTINF:-1 tvg-id="" tvg-country="PH" tvg-language="Tagalog" tvg-logo="" group-title="",CLTV 36
+https://tna5.bozztv.com/cltv36/index.m3u8
```
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply channels/ph.m3u:38]: Add #EXTVLCOPT:http-referrer=https://ssh101.com/

[the remark this answers is not in the document]
Comment (Dum4G) [reply channels/ph.m3u:38]: A refferer was missing. Please retry

File: channels/us.m3u [hunk 5]
```diff
@@ -1910,3 +1922,11 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
 https://thegateway.app/YouToo/YTamerica/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p) [Not 24/7]
 http://188.40.68.167/russia/disney/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="US;LATAM" tvg-language="Spanish" tvg-logo="https://i.imgur.com/BGDFIFt.png" group-title="Sports",Contacto Deportivo
```
Comment (freearhey) [inline channels/us.m3u:1925]: Are you sure that the name of the channel is correct? Here's what I see.
vlcsnap-2021-09-21-16h28m02s490

And google says that Contacto Deportivo is the name of the program
image

Comment (Dum4G) [reply channels/us.m3u:1925]: Hard to tell. It was discovered in Peru channel requests and is taken from here
https://moyobamba.com/contactodeportivo/

Comment (freearhey) [reply channels/us.m3u:1925]: Well, it's probably just a rebroadcast.

File: channels/al.m3u (hunk 2 above)
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/al.m3u:69]: #4655

[the remark this answers is not in the document]
Comment (Dum4G) [reply channels/al.m3u:69]: Loops at different chunk today. Removing

File: channels/ph.m3u (hunk 3 above)
Comment (Dum4G) [reply channels/ph.m3u:35]: @LaneSh4d0w today result is
image

Comment (LaneSh4d0w) [reply channels/ph.m3u:35]: Should work now.

Comment (Dum4G) [reply channels/ph.m3u:35]: image
Finally works

**Review by freearhey — APPROVED** [review]

**Pull request merged**